### PR TITLE
Unity Day Showcase: Rainbow Walk Algorithm

### PR DESCRIPTION
**Pull request by jche1156**
I tinkered around with a toy project so we can show off something on Unity Day! To make it a bit easier for us, I uploaded the project to the site so we don't need to fiddle with as much the day of.

To view:

- Go to Projects Page
- Click the Gold Card labeled "Rainbow Walk"

### CODE REVIEW

File: public/OrbitControls.js
Comment (jche1156) [inline public/OrbitControls.js:1]: This was an external script the author of the random walk script found, to add pan and zoom controls. THe

File: justfile
Comment (jche1156) [inline justfile:1]: I wanted to make it easier for others to install the formatting tools on this project

File: public/rainbowvis.js
Comment (jche1156) [inline public/rainbowvis.js:1]: This adds the ability for us to generate a rainbow gradient.

File: public/random-walk-simulation.js
Comment (jche1156) [inline public/random-walk-simulation.js:1]: I ripped this random walk simulation from the internet, and make a few small tweaks. I changed the path's color, added more stars, then set it up to refresh upon completion, restarting the simulation.

File: src/pages/projects/index.astro
Comment (jche1156) [inline src/pages/projects/index.astro:1]: The project page has a new gold card added to it, leading visitors to the singular project.

**Review by smallnumbers0 — APPROVED** [review]
Nice research

**Pull request merged**